pkg/convert/pot.go

```diff
@@ -0,0 +1,97 @@
+package convert
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+func ToPot(m model.Messages) ([]byte, error) {
+	var b bytes.Buffer
+
+	b.WriteString(fmt.Sprintf("\"Language: %s\n", m.Language))
+
+	for _, message := range m.Messages {
+		if message.Description != "" {
+			_, err := fmt.Fprintf(&b, "#. %s\n", message.Description)
+			if err != nil {
+				return nil, fmt.Errorf("write description: %w", err)
+			}
+		}
+
+		if message.Fuzzy {
+			_, err := fmt.Fprint(&b, "#, fuzzy\n")
+			if err != nil {
+				return nil, fmt.Errorf("write fuzzy: %w", err)
+			}
+		}
+
+		if strings.HasSuffix(message.ID, "\\n") {
```

Comment:
can id really be multiline?

Reply:
I guess, it can

Reply:
Yes, msgid in a PO file can be a multiline string
<img width="741" alt="image" src="https://user-images.githubusercontent.com/94634373/221176132-72af0e33-1f11-4ab0-8ad8-11fa657a3e08.png">